api/v1alpha1/kogitoserverlessworkflow_types.go

```diff
@@ -308,20 +309,31 @@ type State struct {
 	Metadata            *[]Metadata      `json:"metadata,omitempty"`
 }
 
-// KogitoServerlessWorkflowSpec defines the desired state of KogitoServerlessWorkflow
-type KogitoServerlessWorkflowSpec struct {
-	Constants   []Constant   `json:"conditions,omitempty"`
-	Secrets     *[]v1.Secret `json:"secrets,omitempty"`
-	Start       string       `json:"start"`
-	Timeouts    []Timeout    `json:"timeouts,omitempty"`
-	Errors      []Error      `json:"errors,omitempty"`
-	KeepActive  bool         `json:"keepActive,omitempty"`
-	Auth        Auth         `json:"auth,omitempty"`
-	Events      *[]Event     `json:"events,omitempty"`
-	Functions   []Function   `json:"functions,omitempty"`
-	AutoRetries bool         `json:"autoRetries,omitempty"`
-	Retries     Retry        `json:"retries,omitempty"`
-	States      []State      `json:"states"`
+// KogitoSWFSpec defines the desired state of KogitoServerlessWorkflow
+type KogitoSWFSpec struct {
```

Comment:
Why changing the name here? `KogitoServerlessWorkflowSpec` seems way better.

Reply:
@ricardozanini I'd leaved the habit of long names with SpringFrameworkAndHisLongNamesForContextAbstractFactoryBeansAndWhatever  when I've encountered KieSpec in the KieCloudOperator, but I can revert this habit :)